app/views/user/_tracks_progress.erb

```diff
@@ -0,0 +1,22 @@
+<% if user_tracks_summary.any? %>
+  <h3> Progess </h3>
+  <% if params[:all] == 'true' %>
```

Comment:
Maybe in a second iteration, we can allow the user to decide if he want the default to be expanded, just throwing ideas.